### ISSUE TITLE
Add Orientation Auto-fix Support

### ISSUE THREAD

**Issue by baskarap** [gojek/darkroom#14]
Currently, darkroom doesn't read the image exif data and when an image is rotated, darkroom doesn't fix the orientation of it. I am suggesting that darkroom should check the orientation of the image based on the exif data and return as a straight image, given the auto=compress parameter being sent.

Some example on how imgix handles this:
- http://gojek-bike.imgix.net/go_jek/image_uploader_service/image/d3bc45f5-c6cf-4f25-9875-3e918bb2996b_Mochammad_arif_nugraha.jpg?auto=compress
vs
- http://gojek-bike.imgix.net/go_jek/image_uploader_service/image/d3bc45f5-c6cf-4f25-9875-3e918bb2996b_Mochammad_arif_nugraha.jpg

**Comment by ajatprabha** [comment]
@baskarap Both images look same to me orientation wise, are sure the links are correct?

**Comment by baskarap** [comment]
@ajatprabha if you try to load the image in a place where the browser or platform (like slack) doesn't recognise the image orientation, it will be shown differently.

**Comment by ajatprabha** [comment]
Okay, verified. Should be easier after #9 is merged.

**Comment by adalberht** [comment]
Will pick up this issue

**Comment by adalberht** [comment]
Closing this as #16  is merged.
Note that darkroom won't fix the orientation and serve the image as it is if `?auto=compress` query params is not given in the url
